### DISCUSSION TITLE
localhost just doesn't verify

### DISCUSSION THREAD

**Discussion by perler** [coollabsio/coolify#1947] [Q&A]
are there any logs for the verification process of new servers? This is the most basic setup,, coolify on a server and I try to add localhost with a pregenerated keypair without password. I can login to the server with this private key (ssh -i ./id_rsa localhost), so, I just don't see what could possibly wrong here?

**Comment by CodeCooler** [comment]
Any solution to this? I'm having the same issue. I have a feeling it has something to do with firewall settings, as it only started after we began to lock down the machine, but I've opened all the ports mentioned in the docs (8000, 6001, 443 and 22) to my IP, but no luck.

**Reply by peaklabs-dev** [reply]
@CodeCooler Did you change the SSH Port or use a different username?

**Reply by CodeCooler** [reply]
No, but it definitely has something to do with the firewall (iptables running on the same host and managed via ufw): Reopening the ports did nothing to help, but upon turning the firewall off entirely, everything went right back to running and the verification succeeded.

**Reply by peaklabs-dev** [reply]
@CodeCooler So this can be closed, right? We will be adding a firewall management feature to coolfiy soon.

**Reply by CodeCooler** [reply]
Oh nice; built-in firewall management would be great. Thanks, I think my question is answered, but I can't speak for @perler's original post.
